tests/system/providers/amazon/aws/example_emr_serverless.py

```diff
@@ -100,13 +101,14 @@
         configuration_overrides=SPARK_CONFIGURATION_OVERRIDES,
     )
     # [END howto_operator_emr_serverless_start_job]
-    start_job.waiter_check_interval_seconds = 10
+    start_job.wait_for_completion = False
 
     # [START howto_sensor_emr_serverless_job]
     wait_for_job = EmrServerlessJobSensor(
         task_id="wait_for_job",
         application_id=emr_serverless_app_id,
         job_run_id=start_job.output,
+        target_states=EmrServerlessHook.JOB_INTERMEDIATE_STATES,
```

Comment:
we don't want to wait for the end of the job so that we can force-stop it in the next step

(btw this makes the test faster to run because we don't have to wait for the end of the run anymore 😄)

Reply:
I think it's nice to have it in the doc example ? It's pretty self-explanatory (but not necessary indeed).
I'd think it's a good way to highlight some params to put them in the example... But maybe some users just copy paste this and try to adapt it to their needs, in which case it could be confusing ? idk

Reply:
Yep, that's also another way to see it :)

Reply:
I land on the side of this being confusing. I think most folks would assume that the test is running the jobs to completion unless otherwise stated, but it's helpful to know that they can provide states, so maybe just add a short comment above this line to explain what's happening here?

Reply:
ok, done, also changed the state we wait on to be something more "real life"